src/datasources/cache/cache.first.data.source.spec.ts

```diff
@@ -149,4 +149,48 @@ describe('CacheFirstDataSource', () => {
 
     expect(mockCache.set).toHaveBeenCalledWith(cacheDir, expect.anything(), 30);
```

Comment:
We should not use 30 after this change.

Reply:
Changed in [4e30dc6](https://github.com/safe-global/safe-client-gateway-nest/pull/578/commits/4e30dc61d72b12d7b8baa789be526cca80bca083)